docs/architecture/modern-web-apps-azure/choose-between-traditional-web-and-single-page-apps.md

```diff
@@ -35,7 +35,9 @@ Improvements in user experience made possible by the SPA approach must be weighe
 
 ## Blazor
 
-ASP.NET Core 3.0 introduces a new model for building rich, interactive, and composable UI called Blazor. Blazor server-side allows developers to build UI with Razor on the server and for this code to be delivered to the browser and executed client-side using [WebAssembly](https://webassembly.org/). Blazor server-side is available now with ASP.NET Core 3.0 or later. Blazor client-side should be available in 2020.
+ASP.NET Core 3.0 introduces a new model for building rich, interactive, and composable UI called Blazor. Blazor server-side allows developers to build UI with C# and Razor on the server and for the UI to be interactively connected to the browser in real-time using a persistent SignalR connection.
```

Comment:
FYI for future reference (or a patch PR) ... Use "Blazor Server" ... "Blazor server-side" is an older product name that we don't user any longer.